scripts/fuzz_opt.py

```diff
@@ -187,14 +187,16 @@ def numbers_are_close_enough(x, y):
     # float() on the strings will handle many minor differences, like
     # float('1.0') == float('1') , float('inf') == float('Infinity'), etc.
     try:
-        return float(x) == float(y)
+        fx = float(x)
+        fy = float(y)
+        return fx == fy
```

Comment:
Why this change?

Reply:
It can help when debugging as then it an exception is thrown you can see on which line it happens. But maybe not worth it just for that, as it is less clear, I'll remove it.